lib/actions/form.js

```diff
@@ -37,12 +38,18 @@ export function setTime (payload) {
   }
 }
 
+let debouncedPlanTrip  // store as global, so it can be reused.  Maybe there's a better way?
```

Comment:
@trevorgerhardt let me now if there is a better way to do this.  Since a new function is returned on each formChanged event I put the debouncedPlanTrip variable in this scope so it would be available in subsequent calls.